react_app/src/components/Sidebar/Sidebar.js

```diff
@@ -3,12 +3,21 @@ import RangeSlider from 'react-range-slider-input';
 import { Form, FormGroup, Button, Row, Col } from 'react-bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import "react-range-slider-input/dist/style.css";
+import useFetchAuthentication from '../../hooks/useFetchAuthentication'
+
+import Cart from '../../pages/Map/components/Cart'
```

Comment:
The `Cart` is a component of the `Sidebar`, so it's better to keep the code closer to the `Sidebar`. But I think might be better to isolate all components in the root folder with components.